src/main/scala/com/typesafe/webdriver/HtmlUnitWebDriverCommands.scala

```diff
@@ -16,13 +16,15 @@ class HtmlUnitWebDriverCommands() extends WebDriverCommands {
 
   import scala.concurrent.ExecutionContext.Implicits.global
 
-  override def createSession(): Future[String] = {
+
+  override def createSession(desiredCapabilities: JsObject = JsObject(),
+                             requiredCapabilities: JsObject = JsObject()): Future[(String, Either[WebDriverError, JsValue])] = {
```

Comment:
It would probably benefit from being a case classe. I haven't been able to make the session creation fail on phantomjs (even specifying unmet capabilities will result in a successful session creation) hence the current signature. I am not sure whether we should leave it like this or change it to Either[WebDriverError, (String,JsValue)]
That last signature would then easily be modeled as Either[WebDriverError, SessionCreated] with SessionCreated(sessionId:String, capabilities:JsValue)

would you mind if I make that change in a further PR ? this is going to generate quite a few merge conflicts :(